toolchains/mi250.hipcc.adastra.spack/toolchain.cmake

```diff
@@ -33,6 +33,7 @@ set(CMAKE_CXX_FLAGS_INIT "${CMAKE_CXX_FLAGS_INIT} -Wall -Wextra -Wpedantic -Wcas
 
 # FIXME: blas are not properly detected.
 set(CMAKE_CXX_FLAGS_INIT "${CMAKE_CXX_FLAGS_INIT} -isystem $ENV{CRAY_LIBSCI_PREFIX}/include")
+set(CMAKE_EXE_LINKER_FLAGS_INIT "$ENV{PE_MPICH_GTL_DIR_amd_gfx90a} $ENV{PE_MPICH_GTL_LIBS_amd_gfx90a}")
```

Comment:
This is due to not using Cray wrappers. The wrappers used to implicitly inject this dependency.

Reply:
I did not understand, what do you recommend ?
- `CC --cray-print-opts=libs` or
- `$ENV{PE_MPICH_GTL_DIR_amd_gfx90a} $ENV{PE_MPICH_GTL_LIBS_amd_gfx90a}`